opentibiabr/otservbr-global
# Fix Scarlett error on distro, Leiden mechanics and some corpses

# Description
- Fix names of the corpse items from Grave Danger Bosses (according to global, there's only one sprite to all of the GD Bosses, ID 31599 and it's decays, and it's called "dead lichwarrior";

- Fix monster corpse of Lost Exile (it was wrong on the datapack, according to global, it's the same corpse as Lost Husher);

- Fix script from Leiden boss mechanic ("evaporate" from Liquor Spirit should not harm players, should deal damage to Leiden, and heal Ravenous Hunger). Also, fix spawn from ravenous hunger after leiden is killed;

- Fix Scarlett Etzel's script. When a player dies while doing the mechanics, there was an error on distro. This was fixed by an attackerid check;

- Fix Fluids with wrong IDs;

- Deleted duplicate file for POI entrance;

- Fix on Convince Creature Rune target;

- Fix Swiftness imbuement bonus speed, and list of boots that can be imbued according to global;

## Fixes

Resolves #609;
Resolves #608;
Resolves #606;
Resolves #570;
Resolves #289;
Resolves #290;
Resolves #294;
Resolves #344;

## How to test this PR:
1 - Create the monster Lost Exile and kill it, after this PR it has a corpse.

2 - Create the monster "Leiden" and the monster "Liquor Spirit". Kill liquor spirit. It's death should trigger spell Evaporate, and deal damage to Leiden. Also, Evaporate should not deal damage to players.

3 - On Scarlett Boss, center the mirrors, click use on the armor at the room, and hit Scarlett standing in the boss range. When you die from this damage, will not appear any error on distro.

4 - goto /pos 32791, 32334, 9, and use a vial of blood on the grave stone. You should be teleported.

5 - goto /pos 32800, 32339, 11 and use a vial of oil on the lever. The lever should activate.

6 - create a rotworm and use convince creature in it. Should have no error on distro.

7 - Imbuing boots with Swiftness, for example Leather Boots (list of boots that can be imbued was changed according to global). Bonus speed now should be right, +10/+15/+30 speed.


## Type of change

Please delete options that are not relevant.
